pkg/client/unversioned/remotecommand/remotecommand.go

```diff
@@ -155,18 +156,16 @@ func (e *Streamer) doStream() error {
 	}
 	defer conn.Close()
 
-	doneChan := make(chan struct{}, 2)
 	errorChan := make(chan error)
+	var wg sync.WaitGroup
 
 	cp := func(s string, dst io.Writer, src io.Reader) {
+		defer wg.Done()
```

Comment:
I did not run into that stdin issue with the unit tests, see the two commits in this PR touching this file.

Or does that problem show up somewhere other than unit/integration tests?